pkg/netpol/eval/internal/k8s/ipBlock.go

```diff
@@ -23,15 +24,25 @@ type IPBlock struct {
 	ipRange CanonicalIntervalSet
 }
 
-// ToIPRanges returns a string if the ip ranges in the current IPBlock object
+// ToIPRanges returns a string of the ip ranges in the current IPBlock object
 func (b *IPBlock) ToIPRanges() string {
```

Comment:
I wonder if there might be a struct and methods you can reuse from [netip](https://pkg.go.dev/net/netip) instead of roll your own. Not sure if they overlap fully or not (eg can any and all blocks be represented by a network.

Reply:
Will consider in a separate issue to use ip-block representation from netip, and convert to ranges only for computation of disjoint ip-blocks.